source/ui/data/VerticalDelegate.js

```diff
@@ -191,7 +207,8 @@
 			metrics.height = this.pageHeight(list, page);
 			metrics.width  = this.pageWidth(list, page);
 			// update the childSize value now that we have measurements
-			this.childSize(list);
+			var oldChildSize = list.childSize;
+			list.isChildSizeUpdated = (oldChildSize == this.childSize(list)) ? false : true;
```

Comment:
Yes, good advise. But childSize can be updated after metrics is updated.
So, we can update childSize from end of generatePage which metric update is occurred.
Let me find another way to reduce unnecessary childSize() call.